plugins/usd/module/vtkF3DFaceVaryingPointDispatcher.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * @class vtkF3DFaceVaryingPointDispatcher
+ * @brief dispatch face-varying attributes by duplicating points.
+ *
+ * This filter processes arrays on point data in case some of them are flagged
+ * as face-varying in which case points must be duplicated before rendering
```

Comment:
please add that the a face-varying input may not have the same number of tuples in their point data as there is points.